installer.nsh

```diff
@@ -1,44 +1,183 @@
 !define PATH_REMOTE_DIR "C:\Program Files\remoteit-bin"
 !define REMOTE_CLI_EXE "${PATH_REMOTE_DIR}\remoteit.exe"
-
+ 
 !macro customInstall
+    Var /GLOBAL installLog
+    IfFileExists "$TEMP\remoteit.log" file_found file_not_found
+    file_found:
+        FileOpen $installLog "$TEMP\remoteit.log" a 
```

Comment:
Just a question. Where is the $TEMP directory?
And if it's not in one of our current log folders could you please add it to the #Troubleshooting section of the readme.md? Thanks!

Reply:
`ctrl+r  -> %temp%  remoteit.log`

`C:\Users\$USER\AppData\Local\Temp`